tests/tests_neos/test_dastcom5.py

```diff
@@ -124,3 +124,11 @@ def test_download_dastcom5_downloads_file(mock_request, mock_isdir, mock_zipfile
         os.path.join(dastcom5.POLIASTRO_LOCAL_PATH, "dastcom5.zip"),
         dastcom5._show_download_progress,
     )
+
+
+def test_issue_902():
```

Comment:
I propose that we create a mock for `string_record_from_name`, fake the return value to the result that it gets for `"1P"`, and make sure that the output of `record_from_name("1P")` is correct. What do you think?

Reply:
Sure!

Reply:
The return string from `string_record_from_name` is of hundreds of line, I don't think we can fake the return value. Suggestions, please!

Reply:
True, but we don't need all that. Let's do:

```
mock_string_record_from_name.return_value = [
    "90000001 Halley",
    "90000002 Halley",
]
```

And check that the list of 2 integers returned by `record_from_name` contains the correct integers.